slugbuilder/builder/build.sh

```diff
@@ -92,21 +92,23 @@ export CURL_TIMEOUT=600
 
 ## Buildpack detection
 
+# Ordering here is in line number order from buildpacks.txt
 buildpacks=(${buildpack_root}/*)
 selected_buildpack=
 
 if [[ -n "${BUILDPACK_URL}" ]]; then
   echo_title "Fetching custom buildpack"
 
-  buildpack="${buildpack_root}/custom"
+  buildpack="${buildpack_root}/000000*"
```

Comment:
install-buildpack now takes an order instead of a name as parameter 3, and prefixes the name with that ordering component. To get back the custom buildpack, we need to search for the one with that prefix (rather than assuming its the whole of the name).

I suppose I could use "custom*" instead of "000000*", as it looks like the custom one is the only one invoked if its set. Either way, the `*` would need to remain.